hyprshot

```diff
@@ -168,6 +169,7 @@ function begin_grab() {
 	    geometry=`trim "${geometry}"`
             ;;
     esac
+    sleep ${DELAY:-0}
```

Comment:
Better to check if it's a number bigger than 0, and only call sleep if that is the case:

```suggestion
    if [ ${DELAY} -gt 0 ] 2>/dev/null; then
        sleep ${DELAY}
    fi
```